src/Files.App/Helpers/Dialog/DynamicDialogFactory.cs

```diff
@@ -104,7 +104,7 @@ public static DynamicDialog GetFor_RenameDialog()
 
 			dialog = new DynamicDialog(new DynamicDialogViewModel()
 			{
-				TitleText = "EnterAnItemName".GetLocalizedResource(),
+				TitleText = string.Format("CreateNewItemTitle".GetLocalizedResource(), itemType.ToLower()),
```

Comment:
`itemType` wouldn't be localized.

Reply:
Do we have a localized version?

Reply:
This should be resolved in [8cc7298](https://github.com/files-community/Files/pull/16317/commits/8cc72985d87f9a1e173098158bb96d1cdd8c735a)